drivers/hmis/app/models/hmis/access_control.rb

```diff
@@ -26,7 +26,7 @@ class Hmis::AccessControl < ApplicationRecord
   # User, UserGroup, Role, AccessGroup
   scope :filtered, ->(filter_params) do
     return current_scope unless filter_params
-    return current_scope if filter_params[:user_id].blank? && filter_params[:user_group_id].blank? && filter_params[:role_id].blank? && filter_params[:collection_id].blank?
+    return current_scope if filter_params[:user_id].blank? && filter_params[:user_group_id].blank? && filter_params[:role_id].blank? && filter_params[:access_group_id].blank?
```

Comment:
Ugh, thanks for catching that.  I knew something was off at some point but never tracked it down.  Some day we should align the back-end language.

Reply:
yeah for sure. iceboxed it https://github.com/open-path/Green-River/issues/5992